Core/EventHandlers.lua

```diff
@@ -2071,6 +2077,25 @@ end
 
 local IsQuestFlaggedCompleted = C_QuestLog.IsQuestFlaggedCompleted
 
+function Rarity:grabAura()
```

Comment:
This seems somewhat convoluted (and poorly named). I can fix it later, if you prefer?

Reply:
yeah, i guess it can be called something like `OnCOMGOpened` but even then the abbreviation in the function name looks odd. COMG being Chest of Massive Gains that is

Reply:
i guess the newer function name is a little better but can be improved depending on how long you want function names to be. For instance you could list the entire chest name `OnChestofMassiveGainsOpened` but would you consider that to be too long

Reply:
I don't think `OnChestOfMassiveGainsOpened` is too long. Saving on typing with a one-off function name doesn't seem worth it. FWIW, `COMG` is definitely not something anyone would immediately understand :P

Reply:
yeah that could also be changed to something like `OpenedMassiveGainsToday`